client/src/App.tsx

```diff
@@ -17,6 +18,7 @@ function App() {
   const [notifications, setNotifications] = useState<string[]>([]);
   const [gameState, setGameState] = useState<QwixxLogic | null>(null);
   const [gamePath, setGamePath] = useState("");
+  const [availableMoves, setAvailableMoves] = useState<MoveAvailability>({});
```

Comment:
Could the available moves object be apart of game state or does it need to be separate?